dev/io.openliberty.jakarta.persistence.api.3.1/bnd.bnd

```diff
@@ -0,0 +1,22 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
```

Comment:
Should probably add:
```
javac.source: 11
javac.target: 11

Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
```

Reply:
I see the .classpath and org.eclipse.jdt.core.prefs changes, but not the bnd.bnd change